heartbeat/aws-vpc-move-ip

```diff
@@ -154,6 +156,15 @@ Enable enhanced monitoring using AWS API calls to check route table entry
 <shortdesc lang="en">Enhanced Monitoring</shortdesc>
 <content type="boolean" default="${OCF_RESKEY_monapi_default}" />
 </parameter>
+
+<parameter name="eni_id_lookup" required="0">
+<longdesc lang="en">
+Use eni id lookup instead of the default interface id method. This is useful in shared subnet setups.
```

Comment:
Is this supposed to say "instance id" instead of "interface id"? My AWS knowledge is limited; I'm not familiar with "ENI" without looking it up and don't fully understand the situation behind this PR. But based on context -- especially the changes to the `ec2ip_monitor()` function -- it seems that the default would be "instance id".

Reply:
correct, updated